Improve text for IO apps release versions (#633)
vtexdocs/dev-portal-content: docs/vtex-io/App-Guides/vtex-io-documentation-making-your-new-app-version-publicly-available/vtex-io-documentation-releasing-a-new-app-version.md

   - `vtex release major beta` - Releases a **major** beta version.

For more details about the `vtex release` command, check our [VTEX IO CLI reference](https://developers.vtex.com/docs/guides/vtex-io-documentation-vtex-io-cli-command-reference#release).

## Details for each version type

### Major, minor and patch

Following the [SemVer standard](https://semver.org/), the version of an app is composed of three numbers using the format `{major}.{minor}.{patch}`. For instance, if an app has a version `2.13.5`, the **major** number is `2`, the **minor** number is `13` and the **patch** number is `5`.

For each new version of a type, this version type will be increased by one and turn the minor types to `0`, if applicable. For instance, if your app version is `2.13.5` and you want to:

- Release a new **patch**, your new app version will be `2.13.6`.
- Release a new **minor**, your new app version will be `2.14.0`.
- Release a new **major**, your new app version will be `3.0.0`.

Apps should start development with version `0.0.1`. For an app to be publicly available, it must have a major number of at least `1`.

The recommended use cases for each version type are:

> - **Patch**: intended for fixes, small changes to correct errors and bugs.

Reviewer: ```suggestion
- **Patch**: intended for bug fixes and minor issues.
```

- **Minor**: intended for changes in functionality that are not breaking changes like new features, changes in the UI, and others.
- **Major**: intended for breaking changes and changes in the endpoints. Changes in the [Billing Options](https://developers.vtex.com/docs/guides/vtex-io-documentation-billing-options) always require a new **major** release.

### Stable and beta

App versions are created as **stable** by default. **Beta** versions must be explicitly declared and are identified by a dash symbol (`-`) following a sequence of alphanumeric characters after the version number. For instance, `2.13.5` is a **stable** version, and you can have a **beta** version called `2.13.5-beta1`.

The recommended use cases for stable and beta versions are:

> - **Stable**: intended for wide availability and production.

Reviewer: ```suggestion
- **Stable**: intended for wide availability and production environments.
```

> - **Beta**: intended for tests and to limited availability.

Reviewer: ```suggestion
- **Beta**: intended for tests and limited availability.
```

> Observe these points when dealing with a **beta** version:

Reviewer: ```suggestion
Consider the following when dealing with a **beta** version:
```

> 1. **Beta** versions cannot be deployed. If you use `vtex deploy` on a beta version of an app, the accounts and workspaces with the app installed will not update to this version.

Reviewer: ```suggestion
- **Beta** versions cannot be deployed. If you run `vtex deploy` on a beta version of an app, the accounts and workspaces with that specific app version installed will not be updated to this version.
```

> 2. **Beta** versions ignore the `ttl` (time-to-live) parameter set in the [`service.json` file](https://developers.vtex.com/docs/guides/overview-of-vtex-io-services#the-servicejson-file).

Reviewer: ```suggestion
- **Beta** versions ignore the `ttl` (time-to-live) parameter set in the [`service.json` file](https://developers.vtex.com/docs/guides/overview-of-vtex-io-services#the-servicejson-file).
```
